docs/content/ecs/component-generation.md

```diff
@@ -13,12 +13,28 @@ The [code generator]({{urlRoot}}/content/code-generator) uses `.schema` files to
 
 ## Overview
 
-A `struct`, which implements `Unity.Entities.IComponentData` and `Improbable.Gdk.Core.ISpatialComponentData`, is generated for each SpatialOS component. It has the same name as the corresponding schema component.
-The generation process creates a namespace for each struct according to the relevant schemalang component name: `{name of schema component}.Component`
+The code generator generates two structs for each SpatialOS component defined in schema. The generation process places these structs in namespace according to the schemalang component name: `{schema package}.{name of schema component}`. The two structs it generates are:
+
```

Comment:
```suggestion
in a namespace
```
* Think it needs an indefinite article? ("a")?
*

Reply:
`to the schemalang component name` -> what do we mean by that - we have ECS components, SpatialOS components, GameObject components, (and, of course, SpatialOScomponents which are really GameObject components - but I don't want to go there right now!)
Do we mean `the SpatialOS component listed in schemalang`?

Also - REALLY LOVELY active voice. Super clear text. <3 this.